client/src/components/UserSettings/SettingsTagColor.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Collapse, Tag } from 'antd';
+import { BgColorsOutlined } from '@ant-design/icons';
+import { GithubPicker } from 'react-color';
+import { pickerColors, mockedTags as tags, setTagColor, getTagColor, DEFAULT_COLOR } from './userSettingsHandlers';
+import { useLocalStorage } from 'react-use';
+
+const SettingsTagColor: React.FC = () => {
+  const { Panel } = Collapse;
+  const defaultTagColor = JSON.stringify({ default: DEFAULT_COLOR });
+  const [storedTagColors, setStoredTagColors] = useLocalStorage<string>('tagColors', defaultTagColor);
+
+  const collapseTags = (
+    <Collapse accordion ghost>
+      {tags.map((item) => {
+        return (
+          <Panel
+            header={
+              <Tag
+                style={{
+                  cursor: 'pointer',
+                  borderColor: getTagColor(item.name, storedTagColors),
+                  color: getTagColor(item.name, storedTagColors),
+                  backgroundColor: `${getTagColor(item.name, storedTagColors)}10`,
+                }}
```

Comment:
Let's move these 4 lines of the code to it's own function and call `getTagColor` only once

Reply:
Made a new function. It returns an object with styles `borderColor, color, backgroundColor`, it can also accept any other css styles